src/Driver/MySQL/Schema/MySQLIndex.php

```diff
@@ -27,7 +27,11 @@ public static function createInstance(string $table, string $name, array $schema
 
         foreach ($schema as $definition) {
             $index->type = $definition['Non_unique'] ? self::NORMAL : self::UNIQUE;
-            $index->columns[] = $definition['Column_name'];
+            if ($definition['Collation'] === 'D') {
```

Comment:
Ideally, I would like to store values in the table in normalized form. Either with or without DESC, or DESC is another property. This code adds ambiguity to other parts of the index schema.